pkg/api/api.go

```diff
@@ -225,7 +225,7 @@ func (hs *HTTPServer) registerRoutes() {
 	}
 
 	if hs.License.FeatureEnabled("saml") && hs.Features.IsEnabled(featuremgmt.FlagAuthenticationConfigUI) {
-		r.Get("/admin/authentication/", authorize(reqGrafanaAdmin, ac.EvalPermission(ac.ActionSettingsRead, ac.ScopeSettingsAll)), hs.Index)
+		r.Get("/admin/authentication/", authorize(reqGrafanaAdmin, ac.EvalPermission(ac.ActionSettingsRead, ac.ScopeSettingsSAML)), hs.Index)
```

Comment:
Maybe add a FIXME/TODO there for when we have more authentication pages than just SAML :)

Reply:
Good shout, I'll add it in :)